kotest-extensions/kotest-extensions-http/src/jvmMain/kotlin/io/kotest/extensions/http/test.kt

```diff
@@ -3,7 +3,15 @@ package io.kotest.extensions.http
 import io.ktor.client.statement.HttpResponse
 
 suspend fun http(resource: String, f: suspend (HttpResponse) -> Unit) {
+   http(resource, mapOf(), f)
+}
+
+suspend fun http(resource: String, parameters: Map<String, String> = mapOf(), f: suspend (HttpResponse) -> Unit) {
```

Comment:
There's no need to have the default value if you're also providing the overload above.